### PR TITLE
config: update config.yaml for chaos-mesh/chaosd

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
ci stuck in pr https://github.com/chaos-mesh/chaosd/pull/158

### CODE REVIEW

**Comment by WangXiangUSTC** [comment]
@Mini256 @purelind @cwen0 PTAL

**Review by cwen0 — APPROVED** [review]
LGTM

File: prow/config/config.yaml [hunk 1]
```diff
@@ -1574,10 +1574,14 @@ branch-protection:
...
-                  - "pull (unit-test)"
-                  - "pull (integration-test)"
+                  - "ci / pull (amd64, verify) (pull_request)"
+                  - "ci / pull (amd64, build) (pull_request)"
+                  - "ci / pull (amd64, unit-test) (pull_request)"
+                  - "ci / pull (amd64, integration-test) (pull_request)"
+                  - "ci / pull (arm64, verify) (pull_request)"
+                  - "ci / pull (arm64, build) (pull_request)"
+                  - "ci / pull (arm64, unit-test) (pull_request)"
+                  - "ci / pull (arm64, integration-test) (pull_request)"
```
Comment (Mini256) [inline prow/config/config.yaml:1584]: Remove the prefix and suffix of GitHub Action.

```suggestion
                  - "pull (amd64, verify)"
                  - "pull (amd64, build)"
                  - "pull (amd64, unit-test)"
                  - "pull (amd64, integration-test)"
                  - "pull (arm64, verify)"
                  - "pull (arm64, build)"
                  - "pull (arm64, unit-test)"
                  - "pull (arm64, integration-test)"
```

**Review by Mini256 — CHANGES_REQUESTED** [review]

**Review by Mini256 — APPROVED** [review]

**Pull request merged**